packages/react/src/date-picker/DatePickerInput.tsx

```diff
@@ -51,10 +55,20 @@ export const DateInput = forwardRef<HTMLInputElement, DateInputProps>(
 			borderBottomRightRadius: 0,
 		};
 
+		// The secondary label should show todays date in the supplied date format
+		// Fallback to our default date format of `dd/MM/yyyy` if the provided format is not supported
+		// Typescript will complain about incorrect usage, but not everyone relies on typescript
```

Comment:
What does this mean? When will it compain?

Reply:
If you try to pass an invalid `dateFormat` prop, e.g. `<DatePicker dateFormat="MM/ddddd/yyyy" />` typescript will complain because that is not one of the accepted formats.

This just prevents the component from blowing up in that scenario. This could be removed, if it's too defensive.